### PR TITLE
feat(logo): integrate sizes and related pulse as an option for headlines

### PR DESCRIPTION
**Pull request by mfranzke**
- Added `logo` component with `sizes` property, as this leads to the pulse size.
- Added `pulse` property to headline.

### CODE REVIEW

**Review by nmerget — DISMISSED** [review]

**Comment by nmerget** [comment]
Can you add the logo to the angular showcase and add a cypress test for all the showcases https://github.com/db-ui/elements/blob/main/e2e/cypress/integration/99-1-showcases/showcases-other-elements.spec.js

**Comment by mfranzke** [comment]
> Can you add the logo to the angular showcase and add a cypress test for all the showcases https://github.com/db-ui/elements/blob/main/e2e/cypress/integration/99-1-showcases/showcases-other-elements.spec.js

done

File: showcase/vue-showcase/src/components/other-elements/OtherElements.vue [hunk 1]
```diff
@@ -17,6 +18,8 @@ import {
     <DbProgress :value="60" :max="100" />
     <p>DbTag:</p>
     <DbTag>default</DbTag>
+    <p>DbLogo:</p>
+    <DbLogo></DbLogo>
     <DbTag variant="informative">informative</DbTag>
```
Comment (nmerget) [inline showcase/vue-showcase/src/components/other-elements/OtherElements.vue:23]: image
Guess this is wrong copy&paste here

**Review by nmerget — APPROVED** [review]

**Comment by mfranzke** [comment]
@nmerget fixed that aspect, thanks for the catch. Could you please reapprove ?

**Review by nmerget — APPROVED** [review]

**Pull request merged**